mmv1/products/dlp/DeidentifyTemplate.yaml

```diff
@@ -1573,6 +1692,12 @@ properties:
                               description: |
                                 Words or phrases defining the dictionary. The dictionary must contain at least one phrase and every phrase must contain at least 2 characters that are letters or digits.
                               item_type: Api::Type::String
+                    - !ruby/object:Api::Type::Boolean
+                      name: 'replaceWithInfoTypeConfig'
```

Comment:
While running the tests, I saw error for boolean and tried to pass empty object, that worked but further I received API Error stating `Error creating DeidentifyTemplate: googleapi: Error 400: replace_with_info_type_config is not valid as a primitive_transformations. It is only compatible with info_type_transformations.`

Hence, removing this field and respective added test case.